CHANGELOG.md

```diff
@@ -8,6 +8,10 @@
 * [#6100](https://github.com/rubocop-hq/rubocop/issues/6100): Fix a false positive in `Naming/ConstantName` cop when rhs is a conditional expression. ([@tatsuyafw][])
 * [#6526](https://github.com/rubocop-hq/rubocop/issues/6526): Fix a wrong line highlight in `Lint/ShadowedException` cop. ([@tatsuyafw][])
 
+### New features
```

Comment:
[nitpick] Can you move `New fetures` section above `Bug fixes` section?

Reply:
Done